src/cljs/harja/views/urakka/pot2/materiaalikirjasto.cljs

```diff
@@ -113,7 +114,8 @@
      (when-not ei-loytynyt?
        [:div
         [kentat/tee-otsikollinen-kentta {:otsikko "Valitse urakka"
-                                         :arvo-atom mk-tiedot/tuontiin-valittu-urakka
+                                         :arvo-atom tuonti-urakka
```

Comment:
Vähän hämäävää, että arvo-atomiin pitää passata tähän ei-atomi. Tosin, tämä on ollut näin ennenkin, eikä sitä ole mielestäni järkevää korjata juuri tässä.

Reply:
Just sama oli oma ajatukseni. Parametrin nimeksi olisi silloin - ja voitaisiin ehkä myöhemmin - vaihtaa "arvo" tmv.